target/linux/ipq806x/base-files/lib/upgrade/platform.sh

```diff
@@ -3,6 +3,10 @@
 PART_NAME=firmware
 REQUIRE_IMAGE_METADATA=1
 
+get_magic_long_vr2600v() {
```

Comment:
why do you need your own get_magic_long function? Everything that is not explained in the commit message (which is completely empty by the way) is considered as mistake by me.